certified-connectors/Dynamics 365 Translation Service/apiProperties.json

```diff
@@ -0,0 +1,49 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "enter your client id",
+          "scopes": [
+            "user_impersonation"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "AzureActiveDirectoryResourceId": "https://lcsapi.lcs.dynamics.com/",
+            "IsOnbehalfofLoginSupported": true
+          },
+          "customParameters": {
+            "loginUri": {
+              "value": "https://login.windows.net"
+            },
+            "tenantId": {
+              "value": "common"
+            },
+            "resourceUri": {
+              "value": "https://lcsapi.lcs.dynamics.com/"
+            }
+          }
+        }
+      },
+      "token:TenantId": {
+        "type": "string",
+        "metadata": {
+          "sourceType": "AzureActiveDirectoryTenant"
+        },
+        "uiDefinition": {
+          "constraints": {
+            "required": "false",
+            "hidden": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#2266e3",
+    "capabilities": [],
+    "publisher": "Microsoft Corporation"
```

Comment:
@joshftb stackOwner property is missing. Please add it. You can refer to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json) for more details.

Reply:
@v-EgorKozhadei , done with this and the above. Let me know if any further action is required from my side.